src/utils.ts

```diff
@@ -84,4 +86,20 @@ export function replaceTemplateName(dsConfig: IDataSourceConfig, dsName: string,
 	});
 
 	return dsConfigNew;
-}
\ No newline at end of file
+}
+
+export function validateName(name: string): string {
+	if (/^[a-z0-9]{4,253}$/.test(name)) {
+		return undefined;
+	} else {
+		return "The entered name does not comply with the naming conventions. [a-z0-9]";
```

Comment:
I think we need to mention that it needs to have a length between 4 and 253 too

Reply:
right